tests/bson/bson-toPHP_error-005.phpt

```diff
@@ -11,20 +11,29 @@ $tests = [
     pack('VCa*xVa*xx', 18, 0x0E, 'foo', 4, 'bar'), // symbol
 ];
 
+ini_set("mongodb.debug", "stdout");
```

Comment:
Same comment as the test below. At least the title should be changed and the `ini_set()` can possibly be replaced with `--INI--`.

Reply:
It can, but then there as so many other messages that we don't care about. This is better.